src/components/tabs/Tabs.vue

```diff
@@ -2,8 +2,8 @@
   <div>
     <div class="aui-tabs horizontal-tabs">
       <ul class="tabs-menu">
-        <li ref="item" class="menu-item" v-for="tab in tabs">
-          <a :href="'#' + tab.componentOptions.propsData.tabId">{{tab.data.attrs.name}}</a>
+        <li class="menu-item" :key="tab.hash" v-for="tab in tabs">
```

Comment:
If we are changing the order of the attributes please follow the guideline https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended

In this case `v-for` then `key` then `class`

Reply:
Please check my last commit a6357b2

Reply:
Thanks a lot for your review, very useful.
I have tried to fix all issues in my last commit.
Let me know if everything is now ok for merging.
I am also relatively new to github forking and pull requests. 
I hope that I have done things the right way.
Have a nice day and I wish you a very good 2019 year! ;-)
Cheers,
Guillaume